1_fetch/src/my_happy_script.R

```diff
@@ -1,96 +1,133 @@
-
+## ---------------------------
+##
+## Script name: the happy script
+##
+## Purpose of script:
+##
+## Author: Jack Holbrook (USGS)
+##
+## ---------------------------
+## Notes:
+##     ~this is a placeholder for new comit to strucutre
+## ---------------------------
+
+setwd("C:/Users/jholbrook/Documents/Rprojects/targets-1")    ## Jacks's working directory (PC)
 library(dplyr)
+library(remotes)
+library(tidyverse)
 library(readr)
 library(stringr)
 library(sbtools)
 library(whisker)
 
-project_output_dir <- 'my_dir'
+project_output_dir <- '3_visualize'
 
 dir.create(project_output_dir)
 
+# Get the data from Science Base
+getCSV <- function(){
+  mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
```

Comment:
This is "fetching" data from Sciencebase. Now that you have a "1_fetch" step, the output of this function should write to "1_fetch/out"